tests/behaviour/steps/qbwriter.py

```diff
@@ -101,8 +106,10 @@ def step_impl(context, cube_name: str):
         QbColumn(
             "Marker",
             NewQbAttribute.from_data(
+                "Marker",
                 "Marker",
                 data["Marker"],
+                [NewQbConcept("Suppressed")],
```

Comment:
Make this not written out manually. e.g. data["Marker"].dropna() and as a list comprenension.

`[NewQbAttribute(value) for value in data["Marker"].dropna()]`